src/sst_file_writer.rs

```diff
@@ -122,6 +121,31 @@ impl<'a> SstFileWriter<'a> {
         }
     }
 
+    /// Adds a Put key with value to currently opened file
```

Comment:
the ` to currently opened file ` is confusing me and should be removed? This just stores the key/value/ts in the db right?